app/assets/stylesheets/desktop/invite-signup.scss

```diff
@@ -0,0 +1,10 @@
+.invites-show {
```

Comment:
The invites styling also feels like something that could easily be split up into a separate PR.

Reply:
but same as I said above: it will be broken until the next PR happens then

Reply:
I think we can merge an invite styling PR right away. From what I can tell, it's sharing some CSS with the login modal, but it doesn't share markup, so we should split it out the same way you've done here, but as a separate PR, we can review and merge more quickly. 

It's clear to me from the screenshots (and a quick local test) that it is fixing a padding problem, and conceptually some separtion between it and login/create account modals makes 100% sense to me.

Reply:
Moved it from this PR to https://github.com/discourse/discourse/pull/24293